src/components/List/tokenList.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+import getToken from '../../lib/tokens';
+import { Router, Redirect, Route } from 'react-router';
+
+export const AddToken = () => {
+  const tokenLocal = localStorage.getItem('token');
+  const [token, setToken] = useState(tokenLocal);
+
+  const handleNewToken = () => {
+    if (!tokenLocal) {
+      const newToken = getToken();
+      localStorage.setItem('token', newToken);
+    }
+    window.location.href = '/list';
+  };
+
+  return (
+    <div>
+      {token}
+      <button onClick={handleNewToken()}></button>
+    </div>
+  );
+};
+/*
+const autentication = (tokenLocal) => {
```

Comment:
Let's delete this commented out code from this file.

☝️ It's better to not leave any commented code in our codebase unless we're going to use it, soon. Which is rarely the case.
That said, if you really don't want to delete commented code, add a comment to it saying what is its purpose. Generally speaking, you don't need commented code at all.